.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -133,13 +136,23 @@ jobs:
       - name: Pull Docker image
         run: |
           !{{ common.pull_docker("${DOCKER_IMAGE}") }}
+          docker pull "${DOCKER_IMAGE}"
```

Comment:
Merge artefact?
```suggestion
```

Reply:
good catch!

Reply:
To be frank, spotted it only because I just did a very similar conflict resolution